torch/_torch_docs.py

```diff
@@ -8945,7 +8945,7 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.argsort,
            r"""
-argsort(input, dim=-1, descending=False) -> LongTensor
+argsort(input, dim=-1, descending=False, stable=True) -> LongTensor
```

Comment:
Make sure to add the `*` designating where keyword-only arguments begin, and while we're here let's use `Tensor` instead of the legacy `LongTensor`.